juniper_official/Chassis/netsvc.py

```diff
@@ -0,0 +1,769 @@
+""" script to determine whether the network services mode is in sync
```

Comment:
So you know anyone, who can review this file from Junos functionality point of view?

Reply:
Naotaka Fujita is probably the best person, and he's already looked at the script. 
I'm not sure he has a GitHub account to comment here. Will ask.